addons/circulation/functions/fnc_treatment.sqf

```diff
@@ -152,7 +152,8 @@ if (vehicle _medic == _medic && {_medicAnim != ""}) then {
 };
 
 // Play a random treatment sound globally if defined
-if (isArray (_config >> "sounds")) then {
+// Don't attempt to play if sounds array is empty
```

Comment:
instead of that, just deleted unused sounds entry from ace medical treatment actions

Reply:
Removing this undoes #232, making the original sound issue reappear.